GliaWidgets/Sources/Coordinators/EngagementCoordinator/EngagementCoordinator.swift

```diff
@@ -197,6 +205,7 @@ extension EngagementCoordinator {
             updateProps: { viewController.props = $0 },
             onError: { [weak self] _ in
                 guard let self else { return }
+                self.environment.log.prefixed(Self.self).info("Show Unexpected error Dialog")
```

Comment:
would it be beneficial to know error of Unexpected error dialog?

Reply:
https://github.com/salemove/ios-sdk-widgets/pull/852#issuecomment-1854455410